css/main.css

```diff
@@ -0,0 +1,137 @@
+h1 {
+    color: #dedede;
+}
+
+p {
+    color: #eeeeee;
+}
+
+/* Navigation */
+
+.navbar {
+    background: rgb(255, 0, 0, 0.7) !important;
+}
+
+.nav-link,
+.navbar-brand {
+    color: #dedede !important;
+    cursor: pointer !important;
+}
+
+.nav-link {
+    margin-right: 1em !important;
+}
+
+.nav-link:hover {
+    background: rgba(0, 0, 0, 0);
+    color: #af0404 !important;
+}
+
+.navbar-collapse {
+    justify-content: flex-end;
+}
+
+.navbar-toggler {
+    background: rgb(255, 0, 0, 0.7) !important;
+}
+
+.header {
+    background-image: url('../images/bkgtop.png');
+    background-attachment: fixed;
+    background-size: cover;
+    background-position: center;
+    width: 100%;
+    height: 850px;
+}
+
+.overlay {
+    position: absolute;
+    min-height: 100%;
+    min-width: 100%;
+    left: 0;
+    top: 0;
+}
+
+.description {
+    position: absolute;
+    top: 30%;
+    margin: auto;
+    padding: 2em;
+    width: 50%;
+}
+
+.description h1 {
+    color: #af0404;
```

Comment:
There's a cool CSS feature called [custom properties - CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties). It's very handy to create the design color palettes.

Reply:
ooo this is very useful